Added radio group documentation (#1436)
IDEMSInternational/parenting-app-ui: documentation/docs/components/radio_group.md

# Radio Group

## Example

> | type       | name                      |parameter_list |
> | ---------  | ------------              |--------- |
> |radio_group |radio_group_par_2_1	     |answer_list: @local.answer_list_2|
> |radio_group |radio_group_par_2_2	     |answer_list: @local.answer_list_3|
> |radio_group |radio_group_par_2_3	     |answer_list: @local.answer_list_4|

Reviewer: I think the example will make more sense to someone seeing this for the first time if you include the rows defining the local variables:

I'm not sure exactly if this syntax works (using \| instead of | when we want to print | as text rather than use it as a separator for the table) so you should check. 

```suggestion
| type       | name              | value                                 |parameter_list |
| ---------  | ------------     | -------------------              |--------- |
| | answer_list_2 | name:name_var_1 \| text:First;name:name_var_2 \| text:Second;name:name_var_3 \| text:Third; name:name_var_4 \| text:Fourth;name:name_var_5 \| text:Fifth; | |
| | answer_list_3 | name:name_var_1 \| text:First \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg;name:name_var_2 \| text:Second \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg;name:name_var_3 \| text: Third \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg; name:name_var_4 \| text:Fourth \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg;name:name_var_5 \| text:Fourth \| image:/plh_images/icons/heart.svg \| image_checked: /plh_images/icons/tick.svg; | |
|radio_group |radio_group_par_2_1 | 	                     |answer_list: @local.answer_list_2|
|radio_group |radio_group_par_2_2 |              	     |answer_list: @local.answer_list_3|
```

![](images/radio_group.png)

[Google Sheet Demo](https://docs.google.com/spreadsheets/d/1qfatsiHKJ8sCBcJ8oqo3InLrjzT_a2Qxvw6RyxVMTxY/edit#gid=569531329)   
[Live Preview Demo](https://plh-global.web.app/template/comp_radio_group)

## Parameters

| Parameter             | Value                  | Description |
| ---------             | -----------            | --------- |
|answer_list            |null (default)          |No buttons presented|
|answer_list            |strings array           |List of buttons with parameters|

> |radio_button_type      |btn_text (default)      |Text only appears on the button|
> |radio_button_type      |btn_square              |Button with icon and text value|
> |radio_button_type      |btn_image_check         |Button image with the check functionality|
> |radio_button_type      |btn_image_nocheck       |Button image without the check functionality|

Reviewer: I think the radio_button_type parameter is no longer in use (and does nothing at the moment). We are now making these changes only by changing the answer_list, as we did in the example spreadsheet together.

```suggestion
|radio_button_type      |btn_text (default)      |To be removed|
|radio_button_type      |btn_square              |To be removed|
|radio_button_type      |btn_image_check         |To be removed|
|radio_button_type      |btn_image_nocheck       |To be removed|
```
